README.md

```diff
@@ -25,27 +25,31 @@ Installation
 
 ###Prerequisites
 
-####Fedora 23
+####Fedora 25
```

Comment:
That's probably the easiest going forward, as I would think most of the commands would remain constant.